### ISSUE TITLE
Unexpected float, decimal output format

### ISSUE THREAD

**Issue by GDA92** [TomWright/dasel#229]
**Discussion?**
While using Dasel to convert a json format to csv containing float, decimal values, I see that the output is wrongly formated.

**Describe the bug**
Unexpected float, decimal output format

**To Reproduce**
**dasel playground - latest version** Arguments :  **-r json -w csv**

**input json :** 
{
 "x": 123456.1,
 "y": 1234567.1
}
**output csv :** 
x,y
123456.1,**1.2345671e+06** 

**Expected behavior**
x,y
123456.1,**1234567.1** > Preserve original number format and do not convert into scientific format (exponential notation)

**Additional context**
I need to convert json containing WGS84 coordinates and scientific notation is not usable. 
I didn't found any option to force the output format. 

Thanks for help.

**Comment by TomWright** [comment]
I expect we'll some form of value mapping for this to work as you expect.

Perhaps a `--convert x,decimal` where `x` is a selector from the output results and `decimal` is a target datatype for it to be converted to.

**Comment by deus101** [comment]
We had a similar issue WGS84 coordinates. But our JSON de/serializer used a float which was our issue. 
So we have a whole bunch of buildings that are a bit  stunded...longitude wise. 

Anyway our decimals were just rounded off. And you don't quite have the same problem here.
JSON as a rule do not write raw numbers (except for indices?), they convert it to a string wheter its floats or double values. 

Which I would guess can be a minor headache for your solution, if its not going into a Excell. 
That .CSV might need a more rigorus pipeline.

**Comment by TomWright** [comment]
Dasel does use a standard JSON encoder here so I'm happy as far as the encoding goes.

An easy solution here would be to use the [format](https://daseldocs.tomwright.me/usage/flags/format) flag to be in complete control of the output. There are functions available to round, convert to strings etc

**Comment by TomWright** [comment]
Hi @GDA92, @deus101,

This is fixed in `v1.27.2`

```bash
~/repos/github.com/TomWright/dasel [master] $ dasel -v                                             
dasel version development-v1.27.2
~/repos/github.com/TomWright/dasel [master] $ echo '{
"x": 123456.1,
"y": 1234567.1
}' | dasel -r json -w csv
x,y
123456.100000,1234567.100000
```
